artiq/frontend/artiq_coremgmt.py

```diff
@@ -73,6 +73,14 @@ def get_argparser():
                          help="key and file whose content to be written to "
                               "core device config")
 
+    subparsers.add_parser("write_ch_name",
+                          help="store channel number and corresponding device "
```

Comment:
Should be plural, no?

Reply:
it depends on the device they have. is it always more than 1 channel?

Reply:
Yes. Usually several dozens.